docs/tutorials/notebooks/BIBD.ipynb

```diff
@@ -0,0 +1,912 @@
+        "id": "tifxXJkAJ7sH"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Next, we need to define what we are looking for. The ‘find’ keyword indicates that the solver should find a value to for that variable. We want to find a set containing sets of crops. Each set of crops is a crop assignment for a farm."
+      ],
+      "metadata": {
+        "id": "F6pOOUzpKEHU"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "`given farms, crops_per_farm, farms_per_crop, overlap: int`\n",
+        "\n",
+        "`given crops new type enum`\n",
+        "\n",
+        "`find crop_assignment: set of set of crops`"
+      ],
+      "metadata": {
+        "id": "54TpE37nKF5A"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%conjure\n",
+        "find crop_assignment: set of set of crops"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "kG-nX4l8KzPt",
+        "outputId": "f5f874d0-88f6-401e-8ccb-6c2e0eaaf0ad"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "{'crop_assignment': []}"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 4
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "With no constraints it produces an empty set for crop assignment."
+      ],
+      "metadata": {
+        "id": "So-pKa8_K61d"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The first, basic, constraints is the number of farms. The number of sets in the crop_assignment set should equal the numbers of farms. `|crop_assignment|` indicates the size of the crop_assignment set. By setting the size equal to the number of farms (after the such that keyword) the solver will only produce solutions where the size of the set is the same as the number of farms. A comma on the end of line indicates that there are more constraints to follow."
+      ],
+      "metadata": {
+        "id": "q3w-ODcTMBbx"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%conjure\n",
+        "find crop_assignment: set of set of crops\n",
+        "\n",
+        "such that |crop_assignment| = farms,"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "pC0XyFeNMKH-",
+        "outputId": "9f612734-60d2-481d-973e-d014cdc18259"
+      },
+      "execution_count": null,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "{'crop_assignment': [['🥔', '🥦', '🍅'],\n",
+              "  ['🥔', '🥕', '🥒'],\n",
+              "  ['🌽', '🥦', '🥒'],\n",
+              "  ['🌽', '🥕', '🍅']]}"
```

Comment:
This solution actually does have the correct number of assigned crops!

I re-ran the model and got something different, so maybe just re-run eerything to force a different answer?